app/frontend/src/features/communities/events/EventForm.test.tsx

```diff
@@ -190,3 +206,33 @@ describe("Event form", () => {
     await assertErrorAlert(errorMessage);
   });
 });
+
+describe("Submitting an offine event", () => {
+  beforeAll(() => {
+    server.listen();
+  });
+
+  afterAll(() => {
+    server.close();
+    jest.useRealTimers();
+  });
+
+  it("should work", async () => {
+    renderForm();
+
+    userEvent.type(screen.getByLabelText(TITLE), "Test event");
+    userEvent.type(screen.getByLabelText(LOCATION), "tes{enter}");
+    userEvent.click(
+      await screen.findByText("test city, test county, test country")
+    );
+    userEvent.type(screen.getByLabelText(EVENT_DETAILS), "sick social!");
+
+    jest.useFakeTimers("modern");
+    jest.setSystemTime(new Date("2021-08-01 00:00"));
```

Comment:
Move to beforeAll on line 211? Else adding a second test in this describe method will still have this system date. Or not yet have this system date if the second test was executed before this test that sets the system date.